pyproject.toml

```diff
@@ -39,10 +39,10 @@ dev = [
     'jinja2',  # for templating
 ]
 charts = [
-    'darkdetect',
-    'plotly',
-    'py3Dmol',
-    'pandas',
+    'darkdetect',  # for dark mode detection
```

Comment:
it would look much nicer if the inline comments were aligned wrt. `#`, e.g.
```python
    'darkdetect',  # for dark mode detection
    'plotly',      # for interactive plots
    'py3Dmol',     # for visualisation of crystal structures
    'pandas',      # for displaying tables in juptyer notebooks
```

ruff doesn't seem to complain.

Reply:
I agree, formatted comments are more readable, but adding a new dependency with a longer name would require manually reformatting all comments, which is not very nice, especially for a long list like in the dev section. Also, having formatted comments in part of the file is not very nice either. Then you'd have to remember to check every change if any of those comments need to be manually formatted. Ruff doesn't seem to care about pyproject.toml at all. For *.py files, `ruff format` reformats the code so that there are 2 spaces before the start of an inline comment, without any formatting. So I would continue to follow the same standard in pyproject.toml. We can think about formatted comments later, but then we need to have a rule for automatic formatting, preferably including pyproject.toml.

Reply:
Sounds fair. The price of manually updating the potential new lines is low, but let's be consistent with .py files then.